operator/v1/types_etcd.go

```diff
@@ -26,7 +26,7 @@ type Etcd struct {
 
 	// +kubebuilder:validation:Required
 	// +required
-	Spec EtcdSpec `json:"spec"`
+	Spec EtcdSpec `json:"spec,omitempty"`
```

Comment:
Why omitempty? This is a required field and I wouldn't expect an `omitempty` on it?